src/tests/base.cpp

```diff
@@ -19,6 +22,25 @@ void TestBase::storeConfig(ConfigNode& config)
 
 TestBase::TestBase(const Context& context) : _context(context) {}
 
+std::string TestBase::extractFilename(const std::string& path)
+{
+    int pos = path.find_last_of('/');
```

Comment:
To be on the safe side, can you handle `string::npos` as return value?

Reply:
Will do.